.github/renovate.json

```diff
@@ -1,19 +1,46 @@
 {
+  "extends": [
+    "config:base"
+  ],
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "enabledManagers": ["custom.regex"],
   "prConcurrentLimit": 50,
-  "prHourlyLimit": 20,
+  "prHourlyLimit": 50,
   "branchConcurrentLimit": 50,
+  "separateMinorPatch": true,
   "labels": ["renovate"],
+  "logLevelRemap": [
+    {
+      "matchMessage": "/^Custom manager fetcher/",
+      "newLogLevel": "info"
+    }
+  ],
   "packageRules": [
     {
+      "matchDatasources": ["docker", "custom.deb1804", "custom.deb2004", "custom.deb2204"],
+      "matchUpdateTypes": [
+        "minor"
+      ],
+      "automerge": false,
+      "enabled": false
+    },
+    {
+      "matchDatasources": ["docker", "custom.deb1804", "custom.deb2004", "custom.deb2204"],
       "matchUpdateTypes": [
-        "minor",
         "patch",
         "pin",
         "digest"
       ],
-      "automerge": false
+      "automerge": false,
+      "enabled": true
+    },
+    {
+      "matchPackageNames": ["moby-runc", "moby-containerd"],
```

Comment:
could you fill out the full list, with the right skeleton, we can add some owner after if we don't know about them

Reply:
I didn't create all the placeholder blocks because multiple components could be assigned to 1 owner. So there could be less blocks.
For example, instead of 
```
   {
      "matchPackageNames": ["componentA"],
      "assignees": ["devinwong"],
    },
   {
      "matchPackageNames": ["componentB"],
      "assignees": ["devinwong"],
    },
   {
      "matchPackageNames": ["componentC"],
      "assignees": ["devinwong"],
    },
```

we can just do
```
   {
      "matchPackageNames": ["componentA, componentB, componentC"],
      "assignees": ["devinwong"],
    },
```

There are 50+ components now.
I can come back to add all the owners once we have a list, with their github IDs. Does that sound good?

Reply:
I see, so the intention would be to broadcast the new feature to all those changing `components.json` and explaining them that we need them to provide us with the github alias and owner list. correct ?

Reply:
Correct. I will follow up with adding the owners to this config.